### PR TITLE
[#234] Empty line is supposed as stop for puzzle description

### PR DESCRIPTION
**Pull request by pnatashap**
1. Empty comment line is supposed to be as end of the puzzle description
2. If comment contains space symbol it is still considered as puzzle description

### CODE REVIEW

**Comment by pnatashap** [comment]
@php-coder  @yegor256 please take a look

There is a separate issue about missing description for multiline puzzles, so haven't touched it yet  https://github.com/cqfn/pdd/issues/224

**Comment by yegor256** [comment]
@rultor merge

File: test/test_source_todo.rb [hunk 1]
```diff
@@ -137,6 +137,36 @@ def test_todo_colon_parsing_multi_line
...
+  end
+
+  # space is needed in test data in the comment
+  # rubocop:disable Layout/TrailingWhitespace
+  def test_todo_colon_parsing_multi_line_with_empty_line_and_space
+    check_valid_puzzle(
+      '
+      // TODO: #46 task description
+      // 
+      //  second line after empty line is a part of the puzzle in case of space exists
```
Comment (php-coder) [inline test/test_source_todo.rb:160]: Line 145 also has a leading space after an empty line, but it isn't part of puzzle text. How this line is different?

Comment (php-coder) [reply test/test_source_todo.rb:160]: Oh, I see it now: on line 159 there is a trailing space.

Could we have a better rules for distinguish one type of comments from another? The current one seems like a subject for future confusions of our users.

/cc @yegor256 WDYT?

**Pull request merged**